contracts/seed/Seed.sol

```diff
@@ -361,16 +366,17 @@ contract Seed {
       * @dev                     Remove address from whitelist.
       * @param buyer             Address which needs to be unwhitelisted
     */
-    function unwhitelist(address buyer) public onlyAdmin isActive {
-        require(permissionedSeed == true, "Seed: module is not whitelisted");
+    function unwhitelist(address buyer) external onlyAdmin {
+        require(!closed, "Seed: should not be closed");
+        require(permissionedSeed == true, "Seed: seed is not whitelisted");
 
         whitelisted[buyer] = false;
     }
 
     /**
       * @dev                     Withdraw funds from the contract
     */
-    function withdraw() public onlyAdmin allowedToWithdraw {
+    function withdraw() external onlyAdmin allowedToWithdraw {
```

Comment:
I would delete `allowedToWithdraw` and move the lines it contains into this funciton

Reply:
yes, good idea. Thanks

Reply:
Done